x-pack/plugins/fleet/server/saved_objects/migrations/to_v7_16_0.ts

```diff
@@ -18,6 +23,15 @@ export const migrateInstallationToV7160: SavedObjectMigrationFn<Installation, In
 ) => {
   const updatedInstallationDoc = installationDoc;
 
+  const DEFAULT_PACKAGES = [
```

Comment:
shouldn't we just remove `to_v7_16_0` migration file, since this change is not going to be deployed to 7.x branch?

Reply:
The 7.16 migrations are applied to any cluster that upgrades from < 7.16 to the current version, so they need to be retained (customers are technically (though not recommended) allowed to upgrade from any 7.x to any 8.x version)